codegen/templates/init_clients.tmpl

```diff
@@ -23,7 +23,7 @@ type Clients struct {
 func CreateClients(
 	config *zanzibar.StaticConfig,
 	gateway *zanzibar.Gateway,
-) *Clients {
+) interface{} {
```

Comment:
This would look like it doesn't work, but it does. We might want to refactor this in the future to be a bit more typesafe.